### PR TITLE
Database

### PR DESCRIPTION
**Pull request by hadeel11**

### CODE REVIEW

File: server/src/database/db_connection.js [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-const Pool = pg.Pool;
-// const {POOL} = require("pg");
-const url = require('url');
-require('dotenv').config();
+const { Pool } = require("pg");
+const url = require("url");
+require("dotenv").config();
 
-if (!process.env.DB_URL) throw new Error('Envinroment: variable DB_URL must be set');
+let DB_URL = process.env.DB_URL;
```
Comment (okarem) [inline server/src/database/db_connection.js:5]: why did you remove the if?

File: server/src/queries/getData.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const dbConnection = require("../database/db_connection");
+
+const getGuestsTable = cb => {
+  console.log('inside the sql query');
+    dbConnection.query("SELECT * FROM guests;", (err, res) => {
+      if (err) console.log(err);
+      if (res) 
+        {console.log('success inside query', res.rows);
```
Comment (okarem) [inline server/src/queries/getData.js]: remove this pah-leez

File: server/src/database/db_connection.js (hunk 1 above)
Comment (hadeel11) [reply server/src/database/db_connection.js:5]: Hi, I removed it because it failed for undefined  process.env.DB_URL
and I replaced it with: 
DB_URL = process.env.TEST_DB_URL;
if (!DB_URL) throw new Error("Enviroment variable DB_URL must be set");

**Review by okarem — APPROVED** [review]
